Resources/Prototypes/Catalog/VendingMachines/robotics.yml

```diff
@@ -3,3 +3,20 @@
   name: Robotech Deluxe
   description: All the fine parts you need in one vending machine!
   spriteName: robotics
+  startingInventory:
+    ClothingOuterCoatLab: 4
+    #roboticist under
+    CableStack: 4
+    Flash: 4
+    PowerCellLargeHigh: 12
+    #proximity sensor
+    #signaller
+    #health anaylzer
+    Scalpel: 2
+    #circular saw
```

Comment:
The prototype ID for this one is `BoneSaw`, pretty sure

Reply:
I'll add that in, wasn't sure if bonesaw was a hand saw rather than the circular saw. Thanks